teams/community-survey.toml

```diff
@@ -2,10 +2,9 @@ name = "community-survey"
 subteam-of = "community"
 
 [people]
-leads = ["nikomatsakis", "rylev"]
+leads = ["rylev"]
 members = [
-    "Manishearth",
-    "nikomatsakis",
```

Comment:
While this reflects the current reality, I'd like to keep @Manishearth and @nikomatsakis on the team if they still want to be.

Reply:
I see @Manishearth has approved the PR. Niko has been involved in this year's survey albeit in a much reduced role. We probably should keep him as a member.

Reply:
I have reinstated Niko :-)